cargo-espflash/Cargo.toml

```
@@ -26,15 +26,15 @@ pkg-fmt = "zip"
 
 [dependencies]
 cargo_metadata = "0.18.0"
-clap = { version = "4.3.*", features = ["derive", "wrap_help"] }
+clap = { version = "4.4.6", features = ["derive", "wrap_help"] }
 env_logger = "0.10.0"
 esp-idf-part = "0.4.1"
-espflash = { version = "=2.0.2", path = "../espflash" }
+espflash = { version = "=2.1.0", path = "../espflash" }
```

```suggestion
espflash = { version = "2.1.0", path = "../espflash" }
```

Let's not pin this please, just in case we end up publishing a patch release of `espflash` that are lib-only changes.

(I think pinning this dependency is just a relic of the pre-2.0 days? I can't really remember why we're doing that now...)